### ISSUE TITLE
Running your model outside the IHME infrastructure

### ISSUE THREAD

**Issue by dmartizzi** [ihmeuw/covid-model-deaths#27]
Dear IHME team, 

I work for an AI company and we are interested in building an AI platform that collects information from several influential epidemiological models that have been used to make forecasts related to COVID-19. Our goal is not to create new models based on yours, but possibly run it on-the-fly from within our platform. 

I had a look at your public repository, and it looks like it's currently not possible to run the model outside the IHME infrastructure. My impression is that the main reason is that required data are stored in csv files on your machines or in your databases. According to your publications, a lot of the data you use is actually pre-processed data from other publicly available datasets (JHU, Google mobility reports), which should allow any user to run your model, provided that the pre-processing methods are included in the package. 

My questions are the following: 
1. Are you planning to share the code to retrieve, and pre-process the data in the future? 
2. Is such code already included in the package? If this is the case, would it be possible to receive some guidance/documentation on how to run the whole pipeline?

All the best, 
Davide Martizzi

**Comment by aflaxman** [comment]
You are correct that this repo does not include the data necessary to reproduce our results.  I hope that the next thing we open up is the code that does the data processing (and the processed data, too).  My colleagues recently estimated that this might be possible in the next two weeks, but I don't know how realistic that is.

Thank you for your interest in this work!

**Comment by dmartizzi** [comment]
Thank you for your prompt reply and for your work! I'll check your repository for future updates. 

Best,
D/

**Comment by anag004** [comment]
I am a part of a student team trying to model the spread of COVID19 in India. We are taking the curve-fitting approach to extrapolate daily deaths. While we have had some success using the CurveFit library, as @dmartizzi mentioned, code for pre-processing and/or hints on how to use the entire pipeline would be immensely helpful. We really appreciate the excellent work you're doing!

**Comment by aflaxman** [comment]
OMG, was it 9 days ago when I said we hoped to have it out in two weeks!  Thanks for your interest in this, and I'm afraid it is likely going to be more than two weeks from when I originally commented.  Your request will help me advocate for prioritizing this, however.  Thanks again.
